__tests__/app.test.js

```diff
@@ -69,6 +69,74 @@ describe('GET/api/reviews', () => {
           });
         });
     });
+
+    test('200 - responds with an array of reviews objects with ascending order', () => {
+      return request(app)
+        .get('/api/reviews/?order=ASC')
+        .expect(200)
+        .then((res) => {
+          
+          expect(res.body.reviews).toBeInstanceOf(Array);
+          expect(res.body.reviews.length).toBeGreaterThan(0);
+          res.body.reviews.forEach((review) => {
+            expect(review).toMatchObject({
+                title: expect.any(String),
+                designer: expect.any(String),
+                owner: expect.any(String),
+                review_body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                comment_count: expect.any(Number)
+              })
+          });
+          expect(res.body.reviews).toBeSortedBy('created_at', {
+            ascending: true
+          });
+        });
+    });
+
+    test('200 - responds with an array of reviews objects of one category and sort_by provided', () => {
+      return request(app)
+        .get('/api/reviews?category=dexterity&sort_by=title')
+        .expect(200)
+        .then((res) => {
+          
+          expect(res.body.reviews).toBeInstanceOf(Array);
+          expect(res.body.reviews.length).toBeGreaterThan(0);
+          res.body.reviews.forEach((review) => {
+            expect(review).toMatchObject({
+                title: expect.any(String),
+                designer: expect.any(String),
+                owner: expect.any(String),
+                review_body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                comment_count: expect.any(Number)
+              })
+          });
+          expect(res.body.reviews).toBeSortedBy('title', {
+            descending: true
+          });
+        });
+    });
+    test('400 - Bad request when pass invalid query', () => {
+      return request(app)
+        .get('/api/reviews?sort_by=lalala')
+        .expect(400)
+        .then((res) => {
+          expect(res.body.msg).toEqual('Bad Request');
+        });
+    });
+
+    test('400 - Bad request when pass invalid category', () => {
+      return request(app)
+        .get('/api/reviews?category=lalala')
+        .expect(400)
+        .then((res) => {
+          expect(res.body.msg).toEqual('category lalala does not exists');
+        });
+    });
+
```

Comment:
Couple more tests id like to see here, what about an invalid order? Also id like to see separate tests for the valid queries. It just shows that you are testing each behaviour separately.